src/lib.rs

```diff
@@ -1916,8 +1917,11 @@ mod sealed {
 /// [`Vec<u8>`]: std::vec::Vec
 /// [`split_at`]: crate::ByteSlice::split_at
 pub unsafe trait ByteSlice: Deref<Target = [u8]> + Sized + self::sealed::Sealed {
+    #[inline]
```

Comment:
Can you move the attribute below the doc comment? (Here and elsewhere in this change)